.travis.yml

```diff
@@ -1,41 +1,48 @@
-sudo: false
 language: python
-cache: pip
 python:
   - "2.7"
   - "3.4"
+  - "3.5"
```

Comment:
Is it required since we already check against python 3.6?

Reply:
Probably not while you still support Python 2, but I have been caught out by new ~~bugs~~ features during the evolution of Python 3.